app/views/layouts/_alert.html.erb

```diff
@@ -0,0 +1,5 @@
+<% flash.each do |type, msg| %>
+  <div class="alert alert-warning" role="alert">
+    <%= msg %>
+  </div>
+<% end %>
```

Comment:
add a line break at the end of each file. and keep this in mind.

Reply:
done